sdk/appservice/mgmt/src/main/java/com/azure/management/appservice/FunctionApp.java

```diff
@@ -325,6 +325,14 @@
              */
             WithCreate withNewStorageAccount(String name, StorageAccountSkuType sku);
 
+            /**
+             * Creates a new storage account to use for the function app.
+             *
+             * @param storageAccount a creatable definition for a new storage account
+             * @return the next stage of the definition
+             */
+            WithCreate withNewStorageAccount(Creatable<StorageAccount> storageAccount);
```

Comment:
Is it required only for new section added in README? Should we plan to cover same scenario for important resource inside Network, Compute, etc. ?

Reply:
Yes, I think we should cover them (they are largely already covered, as in VM, network. But just this one I was trying to do in README is not...).

Reply:
OK. As we put this on README, then it will be significant to our users. I will double check if any missing in Network and Compute first.